definitions/divorce/json/CaseField.json

```diff
@@ -3395,5 +3395,14 @@
     "Label": "Additional information",
     "FieldType": "TextArea",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "23/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ApplyForDecreeAbsolute",
+    "Label": "Do you want to finalise your divorce?",
+    "FieldType": "YesOrNo",
+    "RegularExpression": "^Yes$",
```

Comment:
Should I do it this way?

AC:
```
Validation is handled in the back end to ensure the event can not be completed if No is selected. In this case a message is displayed to the user saying "You have selected 'No'. You must select 'Yes' to apply for Decree Absolute"
```

or I shall I add callback url for aboutEvent? Or is there any other way?

Reply:
Would need to test it and see if this works. Hopefully we can just use this.

Reply:
Dynamic fields to use would be: `${DNApprovalDate}`, `${D8DerivedPetitionerCurrentFullName}` and `${D8DerivedRespondentCurrentName}

Reply:
I have talked to Gavin. We are going to use checkbox instead.